plugins/woocommerce/client/blocks/packages/components/chip/stories/removable-chip.stories.tsx

```diff
@@ -21,13 +22,38 @@ export default {
 	},
 } as Meta< RemovableChipProps >;
 
-const Template: StoryFn< RemovableChipProps > = ( args ) => (
-	<RemovableChip { ...args } />
-);
+const Template: StoryFn< RemovableChipProps > = ( args ) => {
+	const [ isVisible, setIsVisible ] = useState( true );
+	const handleRemove = () => {
+		setIsVisible( false );
+	};
+
+	if ( ! isVisible ) {
+		return <></>;
+	}
```

Comment:
What do you think of allowing the chip to be restored like so?

```suggestion
	if ( ! isVisible ) {
		return (
			<button onClick={ () => setIsVisible( true ) }>
				<em>Chip was removed, click to reset</em>
			</button>
		);
	}
```

Reply:
Sure